UnityClient/Assets/Scripts/TowerDefence/Wave.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
```

Comment:
Please add a `using System;` statement and make this a `SerializableAttribute` rather than `System.SerializeableAttribute`.